data/formats-data.js

```diff
@@ -563,7 +566,7 @@ exports.BattleFormatsData = {
 		tier: "UU",
 	},
 	oddish: {
-		randomBattleMoves: ["gigadrain", "sludgebomb", "synthesis", "sleeppowder", "stunspore", "toxic", "hiddenpowerfire", "leechseed", "dazzlinggleam", "sunnyday"],
+		randomBattleMoves: ["gigadrain", "sludgebomb", "strengthsap", "sleeppowder", "hiddenpowerfire", "solarbeam", "sunnyday"],
```

Comment:
i did forget about synthesis being better in sun here but that attack dropping sounds pretty nice on bulkevio. perhaps add both? or do you still think it will underperform in the latter

Reply:
probably the first one is better, in case somehow it gets placed on the sun team - i'm not too familiar with the randomTeams script

Reply:
fair